canopy/app/src/components/navigation/SideNav.scss

```diff
@@ -45,6 +45,10 @@ $total-nav-width: map-get($nav-tab-spacings, 'padding') * 2 +
     width: 100%;
     background-color: $background-light;
     height: 2px;
+
+    &.bottom {
+      margin-top: auto;
+    }
```

Comment:
This is nitpicky, but does this need to be a separate class? It seems like the nav hr should default to the bottom

Reply:
If I did not put this then the user sapling list starts in the middle. My thought was that we want them to stack from the top unless it is a specific section such as this profile area.